packages/ethereum/ethereum-types/src/consensus/light_client_header.rs

```diff
@@ -36,6 +38,21 @@ pub struct LightClientUpdate {
     pub signature_slot: u64,
 }
 
+impl LightClientUpdate {
+    /// Validates that the branch depths are correct
+    // TODO: Remove this after type safety is added back (#440)
+    pub fn validate_branch_depths(
```

Comment:
maybe should be called `is_valid_branch_depths`. I'd prefer `is_valid` but I understand that maybe we can't validate other fields?

Reply:
Done